### PR TITLE
feat(card view): Adds Card View to demos

### PR DESCRIPTION
**Pull request by jenny-s51**
**Issue**: #2004

### CODE REVIEW

**Comment by redallen** [comment]
Hey Jenny, any idea why I don't see the demos in the preview?
image

**Comment by jenny-s51** [comment]
@redallen I think the preview might be from an older commit

File: packages/patternfly-4/react-core/src/demos/CardView/examples/CardView.md [hunk 1]
```diff
@@ -0,0 +1,555 @@
+---
+title: 'Card View'
```
Comment (redallen) [inline packages/patternfly-4/react-core/src/demos/CardView/examples/CardView.md]: Lowercase "v" to follow sentence case

File: packages/patternfly-4/react-core/src/demos/CardView/examples/CardView.md [hunk 2]
```diff
@@ -0,0 +1,555 @@
...
+import sparkIcon from './camel-spark_200x150.png';
+import swaggerIcon from './camel-swagger-java_200x150.png';
+import azureIcon from './FuseConnector_Icons_AzureServices.png';
+import restIcon from './FuseConnector_Icons_REST.png';
+
+This is a demo that showcases Patternfly cards.
+
+## Examples
+
+```js title=Default-nav isFullscreen
```
Comment (redallen) [inline packages/patternfly-4/react-core/src/demos/CardView/examples/CardView.md]: Maybe a different title than `Default-nav`?

**Comment by mcarrano** [comment]
@jenny-s51 Basically this demo looks great. My only complaint is that none of the toolbar items are functional. Given that this is a React demo, if we are including elements like the bulk selector and/or pagination, I would expect them to be functional and interact with the card grid. Thoughts @tlabaj ?

**Comment by jenny-s51** [comment]
@mcarrano Thank you for your helpful feedback on this! I'd be happy to implement a functional pagination and bulk selector -- that would definitely make sense here. Waiting on confirmation from @tlabaj

**Comment by tlabaj** [comment]
@jenny-s51 Is this still a WIP?  There is an error when loading the demo.

**Comment by jenny-s51** [comment]
@tlabaj I've been working on making toolbar elements functional. I recently finished implementing the filtering feature. @kmcfaul and I have been discussing the best way to approach bulk-select and I've made some progress, but since there are a lot of moving parts to it and it's still a WIP, the demo isn't rendering at the moment 🙂

**Comment by tlabaj** [comment]
Sounds good Jenny!  Can you add `WIP` to the title of the PR.

**Comment by mcarrano** [comment]
@jenny-s51 It's great to see this with the bulk selection and filter working! Just a couple of questions about what I'm seeing. 

When I have mixed selection, I was expecting the bulk selector checkbox to show mixed selection. But when I look at the Bulk Selector demo, I see that mixed selection is not displayed in either case. It seems like that used to work. @tlabaj do you know if anything has changed here? Am I misremembering this? I am also also to get into a state where I select a count of items per page that exceeds the count of items resulting in what you see below. There are only 10 items despite selecting 20 per page.

Screen Shot 2020-03-13 at 3 51 53 PM

Also, a minor nit. The pagination control at the bottom of the page has no padding so when the input field is disabled it looks like the pagination is broken in two. @mcoker should there be padding around the pagination or some layout used here to prevent this?

Screen Shot 2020-03-13 at 3 52 10 PM

**Comment by mcoker** [comment]
@mcarrano I'm looking at https://patternfly-react-pr-3441.surge.sh/documentation/react/demos/cardview/basic and don't see a bottom pagination. Can you link me to the example where you see that?

**Comment by jenny-s51** [comment]
@mcoker it should show up when you scroll all the way down to the bottom 
Screen Shot 2020-03-16 at 12 26 21 PM

**Comment by mcoker** [comment]
@jenny-s51 ah, thanks! the pagination is currently outside of `.pf-c-page`, which won't work. @mcarrano is there a design for how the toolbar/pagination should look for card view? In the core demo, we don't have bottom pagination. https://www.patternfly.org/v4/documentation/core/demos/cardview/basic

Also looks like we're going to need to work on the top toolbar spacing in the react demo.

**Comment by mcarrano** [comment]
Good question @mcoker . I don't think our mocks accounted for that, but they should. @mceledonia @gdoyle any idea what that might look like? Ideally we should have pagination both at the top and bottom of the view. However Card View is kinda unique in that the cards live on a gray background. Would we just add a card or a toolbar to the bottom of the view?

**Comment by jenny-s51** [comment]
@mcarrano @mcoker What is the current status of the toolbar/pagination design? I can open any relevant issues regarding those in the corresponding repos -- just let me know what you need from my end!

Michael Coker is right, the bottom pagination wasn't in the core mocks that I was referencing, so I'd be happy to remove it for now. Would you guys say that's fair game at the moment to get this PR merged?

**Comment by mcarrano** [comment]
@jenny-s51 Let's just remove the bottom pagination for now so we can get this merged. I've opened a new design issue here so we can consider what this should look like: https://github.com/patternfly/patternfly-design/issues/834  Once we have that I can open new issues to update both the core and React demos.

**Comment by jenny-s51** [comment]
@mcarrano Sounds like a good plan -- thank you! I've just updated this PR and the bottom pagination is no longer part of the demo.

File: packages/react-core/src/demos/CardView/examples/CardView.md [hunk 3]
```diff
@@ -0,0 +1,770 @@
...
+PageSectionVariants,
+PageSidebar,
+SkipToContent,
+TextContent,
+Text,
+Toolbar,
+ToolbarGroup,
+ToolbarItem
+} from '@patternfly/react-core';
+// make sure you've installed @patternfly/patternfly
```
Comment (redallen) [inline packages/react-core/src/demos/CardView/examples/CardView.md]: Remove this line: `// make sure you've installed @patternfly/patternfly`

**Pull request merged**